src/css/profile/mobile/common/calendar.less

```diff
@@ -0,0 +1,408 @@
+.calendar_material {
+	position: absolute;
+	display: block;
+	background-color: white;
+	font-size: 14 * @px_base;
+	.font(regular);
+	color: #404040;
+	width: 100%;
+	ul.calendar_line {
+		position: relative;
+		display: block;
+		clear: both;
+		font: inherit;
+		margin: 0;
+		padding: 0;
+		overflow: hidden;
+		width: 100%;
+		li {
+			float: left;
+			position: relative;
+			list-style-type: none;
+			list-style-image: none;
+			text-align: center;
+			vertical-align: middle;
+			font: inherit;
+			cursor: default;
+			overflow: hidden;
+			margin: 0;
+			padding: 0;
+		}
+	}
+	div.calendar_month_cont {
+		position: relative;
+		display: block;
+		width: 100%;
+		height: 36 * @px_base;
+		margin-bottom: 10 * @px_base;
+		overflow: hidden;
+		font: inherit;
+		user-select: text;
+		ul.calendar_line {
+			li.calendar_cell.calendar_month_hdr {
+				width: 100%;
+				height: 36 * @px_base;
+				line-height: 36 * @px_base;
+				font-size: 17 * @px_base;
+				div.calendar_month_arrow {
+					position: absolute;
+					top: 0;
+					width: 36 * @px_base;
+					height: 36 * @px_base;
+					color: inherit;
+					text-align: center;
+					background-position: center center;
+					background-repeat: no-repeat;
+
+					&.calendar_month_arrow_left {
+						left: 0%;
+						mask-image: url("images/4_Dialogs/tw_numberpicker_prev_mtrl.svg");
+						mask-repeat: no-repeat;
+						mask-position: center;
+						mask-size: 100%;
+						background-color: #8e8e8e;
+					}
+					&.calendar_month_arrow_right {
+						right: 1%;
+						mask-image: url("images/4_Dialogs/tw_numberpicker_next_mtrl.svg");
+						mask-repeat: no-repeat;
+						mask-position: center;
+						mask-size: 100%;
+						background-color: #8e8e8e;
+					}
+				}
+				div.calendar_switch {
+					text-decoration: underline;
+				}
+			}
+		}
+	}
+	div.calendar_days_cont {
+		position: relative;
+		display: block;
+		width: 100%;
+		height: 32 * @px_base;
+		margin-bottom: 7 * @px_base;
+		overflow: hidden;
+		-webkit-user-select: text;
+		-khtml-user-select: text;
+		-moz-user-select: text;
+		-ms-user-select: text;
+		-o-user-select: text;
+		user-select: text;
+		ul.calendar_line {
+			height: 31 * @px_base;
+			li {
+				height: 32 * @px_base;
+				line-height: 32 * @px_base;
+				font-size: 11 * @px_base;
+				color: #a6a6a6;
```

Comment:
What's the color value "#a6a6a6"?

Reply:
GUI guide defined.
but 1.7ver, change a6a6a6 to 909090 :(..
I will change.